### ISSUE TITLE
Bottom line artifact stays after command selected

### ISSUE THREAD

**Issue by michalczerwinski** [gui-cs/F7History#13]
Repro:
* open history with F7
* select some entry

Result:
Status line is still displayed:
![image](https://github.com/gui-cs/F7History/assets/1795630/295758f6-2232-4e56-adbd-07f5d5b6044e)

Expected result:
* screen should be restored to original state

**Comment by tig** [comment]
Thanks for the report. This is a known issue in Terminal.Gui that will be fixed probably today. However until `Microsoft.PowerShell.ConsoleGuiTools` picks it up F7History wont see it.

See https://github.com/gui-cs/Terminal.Gui/issues/2634 and https://github.com/PowerShell/GraphicalTools/issues/203

I'll leave this issue open until it's fixed-fixed.

**Comment by tig** [comment]
Fixed in GraphicalTools v0.74. 

Upgrade via `Install-Module Microsoft.PowerShell.ConsoleGuiTools -Force`

**Comment by BDisp** [comment]
@tig I'm still confused about this issue with the F7History which the bottom status bar stays visible after pressing escape. You can see that I've the latest versions as you can see in the video. I've F7History version 1.1.1 but in this repo has the version 1.1.0 in the releases. If I run the `ls | ocgv` command the status bar is cleared after pressing escape.

https://github.com/gui-cs/F7History/assets/13117724/441d9d26-ab74-4916-b240-2b1b26e4f14b

**Comment by tig** [comment]
Weird. I just repo'd this too.

**Comment by tig** [comment]
Reopening.

Here's what I found:

- This does not reproduce in VSCode's terminal.
- Changing WindowsDriver to emit `?1049l/?1049h` instead of `?1047l/?1047h` fixes this on Windows Terminal
- On Windows Terminal, NetDriver works properly, but screws up the oh-my-posh prompt spacing. 
- On Windows Terminal, in WSL, NetDriver works properly, and does NOT screw up the oh-my-posh prompt spacing
- On Windows Terminal, in WSL, CursesDriver works properly.

So, my current thought is to do a PR to Terminal.Gui that changes Windows driver to use `1049h/l`.

I am preparing PRs for F7History and ConsoleGuiTools that enables better identification of what code is running. OCGV will now show the Terminal.Gui version and ConsoleGuiTools version if -Debug or -Verbose are provided. F7History will use `ocgv -Debug` if $Debug=$true. I've also added a `-UseSystemConsole` paramter to `ocgv` and made it so F7History will set this if `$UseSystemConsole=$true`. This will be reverted in Terminal.Gui v2 because `ConfigurationManager` can be used to set it there.
